crates/cli/src/subcommands/sql.rs

```diff
@@ -27,39 +28,31 @@ pub fn cli() -> clap::Command {
                 .conflicts_with("interactive")
                 .help("The SQL query to execute"),
         )
-        .arg(Arg::new("interactive")
-                 .long("interactive")
-                 .action(ArgAction::SetTrue)
-                 .conflicts_with("query")
-                 .help("Runs an interactive command prompt for `SQL` expressions"),)
+        .arg(
+            Arg::new("interactive")
+                .long("interactive")
+                .action(ArgAction::SetTrue)
+                .conflicts_with("query")
+                .help("Runs an interactive command prompt for `SQL` expressions"),
+        )
         .group(
             ArgGroup::new("mode")
-                .args(["interactive","query"])
+                .args(["interactive", "query"])
                 .multiple(false)
-                .required(true)
-        )
-        .arg(
-            Arg::new("identity")
-                .long("identity")
-                .short('i')
-                .conflicts_with("anon_identity")
-                .help("The identity to use for querying the database")
-                .long_help("The identity to use for querying the database. If no identity is provided, the default one will be used."),
+                .required(true),
         )
+        .arg(common_args::identity().conflicts_with("anon_identity").long_help(
+            "The identity to use for querying the database. If no identity is provided, the default one will be used.",
+        ))
         .arg(
             Arg::new("anon_identity")
                 .long("anon-identity")
                 .short('a')
                 .conflicts_with("identity")
                 .action(ArgAction::SetTrue)
-                .help("If this flag is present, no identity will be provided when querying the database")
-        )
-        .arg(
-            Arg::new("server")
-                .long("server")
-                .short('s')
-                .help("The nickname, host name or URL of the server hosting the database"),
```

Comment:
note that this helptext changed by using the `common_args` version. I don't personally feel that this "custom" version is meaningfully more helpful.